opendps/opendps.c

```diff
@@ -247,27 +247,27 @@ set_param_status_t opendps_set_parameter(char *name, char *value)
     }
     return status;
 }
+
 /**
  * @brief      Format Past
  *
- * @param      name   Name of parameter
- * @param      value  Value as a string
- *
- * @return     Status of the operation
+ * @return     True on success, else false
  */
 bool opendps_format_past()
 {
-    
+    if (!past_format(&g_past))
+        return false;
     past_format(&g_past);
```

Comment:
Should this not be removed now?

Reply:
If that is the case it will always return true so we should change this function to be void and not return a bool

Reply:
I don't like the idea of throwing away an error so I'll make it passed back up the chain with the [handle_init()](https://github.com/geekbozu/opendps/blob/779a7cb31b3240f220fac9250e1ed8a5c37791c2/opendps/protocol_handler.c#L130) function

Reply:
I meant more of we are now calling past_format 2 times! but yeah looks much better!